pbjson-test/src/lib.rs

```diff
@@ -32,6 +32,291 @@ pub mod test {
         include!(concat!(env!("OUT_DIR"), "/test.common.rs"));
         include!(concat!(env!("OUT_DIR"), "/test.common.serde.rs"));
     }
+    pub mod skipped_tests {
+        pub mod syntax3 {
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.syntax3.rs"));
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.syntax3.serde.rs"));
+        }
+        pub mod common {
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.common.rs"));
+            include!(concat!(env!("OUT_DIR"), "/skipped/test.common.serde.rs"));
+        }
+    }
+}
+
+#[cfg(test)]
+mod skipped_tests {
+    use super::*;
+    use crate::test::syntax3::kitchen_sink::MixedCase;
+    use chrono::TimeZone;
+    use pbjson_types::{Duration, Timestamp};
+    use test::skipped_tests::syntax3::*;
+
+    #[test]
+    fn test_empty() {
+        let message = Empty {};
+
+        let encoded = serde_json::to_string(&message).unwrap();
+        let _decoded: Empty = serde_json::from_str(&encoded).unwrap();
+
+        let empty = serde_json::from_str::<Empty>("{\n \"foo\": \"bar\"\n}").unwrap();
+        assert_eq!(empty, Empty {});
+    }
+
+    #[test]
+    fn test_kitchen_sink() {
```

Comment:
I think I would prefer we have a set of dedicated tests of this functionality, with their own proto files, tests, etc... rather than reusing the existing tests

Reply:
The reason why I duplicated the tests was because that was the only way I was able to think of that would verify that all the existing functionality won't break with the skip flag was enabled.

Probably something that can work here is having some macro with the test and then pass specific deserializer:

Reply:
One option might be to add a feature flag to pbjson-test and have CI run with it enabled and disabled? :thinking:

It's a bit of a hack, but would work and avoid complicating the tests themselves